src/models/time_series/realmlp/realmlp_predictor.py

```
@@ -0,0 +1,802 @@
+        self._latent_mean: Optional[np.ndarray] = None
+        self._latent_cov_inv: Optional[np.ndarray] = None
+        self._cal_latent: Optional[np.ndarray] = None
+        self._cal_residuals: Optional[np.ndarray] = None
+        self._conformal_alpha: float = 0.1
+        self._conformal_k: int = 50
+
+    def _create_model(self) -> nn.Module:
+        cfg = self.config
+        input_size = cfg.get("input_size")
+        if input_size is None:
+            raise ValueError("Config must include 'input_size'")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)